src/tetris/scene/collectionScene.ts

```diff
@@ -0,0 +1,109 @@
+/// <reference path="../../../definitions/phaser.d.ts"/>
+
+import Vector2 = Phaser.Math.Vector2;
+
+import TextButton from "tetris/ui/textButton";
+import config from "tetris/config";
+import Game from "tetris/game";
+import Brick from "tetris/brick/brick";
+import BrickType from "tetris/brick/brickType";
+import BrickFactory from "tetris/brick/brickFactory";
+import SkinStorage from "tetris/brick/skinStorage";
+import Skin from "tetris/brick/skin";
+import { skinRarityColor } from "tetris/brick/skinRarity";
+
+export default class CollectionScene extends Phaser.Scene {
+
+	//region public members
+	//endregion
+
+	//region public methods
+	public create(): void {
+		this._createBackground();
+		this._createButtons();
+	}
+
+	public update(time: number, delta: number): void {
+		this._pipeline.setFloat1('uTime', time / 2600);
```

Comment:
Why `time / 2600`?
Seems not to be a time conversion.

Reply:
✨ its magic